### DISCUSSION TITLE
Créer une seule fois les images des diagnostics par territoire

### DISCUSSION THREAD

**Discussion by alexisig** [MTES-MCT/sparte#627] [General]
## Situation

Aujourd'hui les images statiques (cartes, graphiques etc.) sont générées pour chaque diagnostic

## Situation souhaitée

Les images statiques identiques sont partagées entre tous les diagnostics du même territoire avec les même paramètres

**Comment by leabouche** [comment]
Soulève la question de la gestion du paramétrage des diags
On pourrait créer des images de diagnostics avec les paramètres "standard" actuellement par défaut, et générer des nouveaux diag ad hoc quand les paramètres sont modifiés.
